index.html

```diff
@@ -0,0 +1,126 @@
+                    <li>Logout</li>
+                </ul>    
+            </nav>
+
+            <nav>
+                <ul class="bottom-nav">
+                    <li><a href="index.html"><img class="logo" src="img/Last Day Events-logos_black.png" alt="LDE logo"></a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>Sponsor</li>
+                    <li>News</li>
+                    <li>LDE Campaign</li>
+                </ul>    
+            </nav>
+        </div>
+
+        <div class="intro">
+            <p class="first-para">"Ethio Art! Great Painters and Poets"</p>
+            <h1 class="first-head">ART GALLERY AND <br> POETRY READING EVENT <br> 2023</h1>
+            <p class="second-para">
+                An event to promote and celebtate works of new and seasoned artists in Ethiopia.
+                Happening for the past ten years the event have been growing with more audience number
+                each year.
+            </p>
+            <h2 class="second-head">2023.02.10(FRI) ~ 12(SUN)</h2>
+            <p class="last-para">@Abrehot Library and Cultural Center</p>
+        </div>
+    </section>
+
+    <main class="main-section">
+        <div class="main-program">
+            <div class="header-main">
+                <h3 class="header-in-main">Main Events</h3>
+                <span class="underline"></span>
+            </div>
+            <div class="main-desk">
+                <div class="program-container">
+                    <img class="program-img" src="img/museum-art-tour-icon-cartoon-real-walk-vector.webp" alt="Gallery Tour">
+                    <h4 class="program-head" >Gallery Tour</h4>
+                    <p class="program-para" >Tour some of the paintings and photographs of the best artists.</p>
+                </div>
+                <div class="program-container">
+                    <img class="program-img" src="img/poetry reading.png" alt="Gallery Tour">
+                    <h4 class="program-head" >Poetry Reading</h4>
+                    <p class="program-para" >The best poets from Addis. Listen to the experience of the writers own words.</p>
+                </div>
+                <div class="program-container">
+                    <img class="program-img" src="img/book reading.png" alt="Gallery Tour">
+                    <h4 class="program-head" >Book Reading</h4>
+                    <p class="program-para" >Classic event in which the writers of the years best sellers introducing their books to you.</p>
+                </div>
+                <div class="program-container">
+                    <img class="program-img" src="img/short play.jpg" alt="Gallery Tour">
+                    <h4 class="program-head" >Short Plays</h4>
+                    <p class="program-para" >Short plays that reveals personal stories and explore complex social issues.</p>
+                </div>
```

Comment:
- [ ] It's time we explore more about [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp). I suggest that we consider the use of [`<article>`](https://www.w3schools.com/html/html5_semantic_elements.asp) HTML tag 👍 . This will help us ensure that we ensure the proper practice of Semantic HTML.